### PR TITLE
Erin/casework model

### PR DESCRIPTION
**Pull request by erinysong**
## casenote model
model for casenotes and relations with staff and participant

### CODE REVIEW

File: app/models/participant.rb [hunk 1]
```diff
@@ -1,4 +1,5 @@
 class Participant < ApplicationRecord
+  has_many: casenote
```
Comment (adowski) [inline app/models/participant.rb]: This should look like `has_many :casenotes`!

File: app/models/staff.rb [hunk 2]
```diff
@@ -1,3 +1,4 @@
 class Staff < ApplicationRecord
+    has_many: casenotes
```
Comment (adowski) [inline app/models/staff.rb]: Same as above, `has_many :casenotes`

File: app/models/casenote.rb [hunk 3]
```diff
@@ -0,0 +1,3 @@
+class Casenote < ApplicationRecord
+  belongs_to :staff
```
Comment (adowski) [inline app/models/casenote.rb]: There should also be a `has_one` relationship with participants here!

File: db/migrate/20191004042135_create_casenotes.rb [hunk 4]
```diff
@@ -0,0 +1,13 @@
+class CreateCasenotes < ActiveRecord::Migration[6.0]
+  def change
+    create_table :casenotes do |t|
+      t.string :description
+      t.boolean :internal
+      t.references :staff, null: false, foreign_key: true
+      t.references :participant, null: false, foreign_key: true
+      t.string :title
```
Comment (kyuwonk) [inline db/migrate/20191004042135_create_casenotes.rb]: add `, null: false`

**Pull request merged**